tools/release-plugin.sh

```diff
@@ -201,8 +249,14 @@ if ! gh run watch "${BUILDID[0]}" --exit-status; then
 	echo "Build failed! Check for build errors on GitHub for more information." && die
 fi
 
-# After this, run tools/create-release-branch.sh to create a release branch.
-yellow "Build is complete. Creating a release branch."
-tools/create-release-branch.sh "$PROJECT" "$VERSION"
+yellow "Build is complete."
+# Run tools/create-release-branch.sh to create a release branch for each project.
+for PREFIX in "${!PREFIXES[@]}"; do
+	git checkout prerelease
+	VERSION="${PREFIXES[$PREFIX]}"
+	PROJECT="$PREFIX"
+	yellow "Creating release branch for $PROJECT $VERSION"
+	tools/create-release-branch.sh "$PROJECT" "$VERSION"
+done
```

Comment:
is this going to do the right thing, or will it wind up confused because the prerelease branches each have extraneous plugins also in the release state? Do we really need to handle the case of releasing more than one different prefix at once?

Reply:
Hmm, how would we do this normally? Occasionally a team will find out we are doing a release and ask us to release it alongside Jetpack, but I haven't tested that flow since we used the prerelease branch.

Can they all just be done from the same release branch? Something like Protect can just be released from `jetpack/branch-12.0`, etc?

Reply:
Looks like CI will be ok after all, now that I look closer for a release branch it only checks the relevant plugins.